en/core-api/credit-card.md

```diff
@@ -200,11 +213,17 @@ curl -X POST \
 #### **PHP**
 If you are using Composer, follow the steps given below.
 1. Install [midtrans-php library](https://github.com/Midtrans/midtrans-php).
-2. `composer require midtrans/midtrans-php`
+
+2. ```bash 
+   composer require midtrans/midtrans-php
+   ```
 
 > Alternatively, if not using Composer, follow the steps given below.
-> 1.  Download [midtrans-php library](https://github.com/Midtrans/midtrans-php/archive/master.zip).
-> 2. `require_once dirname(__FILE__) . '/pathofproject/Midtrans.php';`.
+> 1. Download [midtrans-php library](https://github.com/Midtrans/midtrans-php/archive/master.zip).
+>
+> 2. ```php
+>    require_once dirname(__FILE__) . '/pathofproject/Midtrans.php';
+>    ```
```

Comment:
I don't get it, why would you break down a straight forward installation step into 2 numbered step?
1. it doesn't even make sense, because "2." is the how-to of "1." it is not a separate step
2. you don't do that to Node JS or Python, why inconsistently do that to PHP?

<img width="777" alt="Screen Shot 2020-11-25 at 15 38 27" src="https://user-images.githubusercontent.com/13027142/100202842-6ee15c80-2f34-11eb-9887-917139b4f0ad.png">